RecoHI/HiEvtPlaneAlgos/interface/HiEvtPlaneFlatten.h

```diff
@@ -105,13 +201,62 @@ class HiEvtPlaneFlatten {
     return psi;
   }
   
+  double GetOffsetPsi(double s, double c, double w, uint m,  double vtx, int centbin)
+  {
+    int indx = GetOffsetIndx(centbin,vtx);
+    double snew = s-yoffDB[indx];
+    double cnew = c-xoffDB[indx];
+    double psi = atan2(snew,cnew)/vorder;
+    if((fabs(snew)<1e-4) && (fabs(cnew)<1e-4)) psi = 0.;
+    psi=bounds(psi);
+    psi=bounds2(psi);
+    soff_ = snew;
+    coff_ = cnew;
+    w_ = w;
+    mult_ = m;
+
+    return psi;
+  }
+  
   ~HiEvtPlaneFlatten(){}
   int GetHBins(){return hbins;}
```

Comment:
``` int getHBins() const { return hbins_; } ```
The same applies to all
- methods start with a lower case letter
- methods not changing the state of the object should be declared const
- member variables should be distinguished in some way from local scope variables